sensor.py

```
@@ -0,0 +1,38 @@
+from homeassistant.components.sensor import SensorEntity
+from homeassistant.const import TEMP_CELSIUS
+from homeassistant.helpers.entity import DeviceInfo
+
+from .const import DOMAIN
+
+async def async_setup_entry(hass, entry, async_add_entities):
```

Ruff (0.11.9)

Missing docstring in public function

(D103)